.github/release-template.md

```diff
@@ -0,0 +1 @@
+Check the [v{{.Version}} contrib changelog](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tags/v{{.Version}}) and the [v{{.Version}} core changelog](https://github.com/open-telemetry/opentelemetry-collector/releases/tags/v{{.Version}}) for changelogs on specific components.
```

Comment:
```suggestion
Check the [v{{.Version}} contrib changelog](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v{{.Version}}) and the [v{{.Version}} core changelog](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v{{.Version}}) for changelogs on specific components.
```
I get 404s with the `s`, please check

Reply:
Hello @atoulme, I've made that change. Please check to confirm the 404 error is gone.